packages/@react-spectrum/list/src/ListView.tsx

```diff
@@ -136,6 +155,62 @@ function ListView<T extends object>(props: ListViewProps<T>, ref: DOMRef<HTMLDiv
     // focusable children in the cell.
     focusMode: 'cell'
   }), [state, domRef, direction, collator]);
+
+  let provider = useProvider();
+  let {checkboxProps} = useGridSelectionCheckbox({key: null}, state);
+  let dragState: DraggableCollectionState;
+  if (isListDraggable) {
+    dragState = dragHooks.useDraggableCollectionState({
+      collection: state.collection,
+      selectionManager: state.selectionManager,
+      renderPreview(selectedKeys, draggedKey) {
+        let item = state.collection.getItem(draggedKey);
+        let itemWidth = domRef.current.offsetWidth;
+        let showCheckbox = state.selectionManager.selectionMode !== 'none' && state.selectionManager.selectionBehavior === 'toggle';
+        let isSelected = state.selectionManager.isSelected(item.key);
+        return (
+          <Provider
+            {...provider}
+            UNSAFE_className={classNames(listStyles, 'react-spectrum-ListViewItem', 'is-dragging')}
+            UNSAFE_style={{width: itemWidth, paddingInlineStart: 0}}>
+            <div className={listStyles['react-spectrum-ListViewItem-grid']} data-testid="dragpreview">
+              <div className={listStyles['react-spectrum-ListViewItem-draghandle-container']}>
+                <div className={listStyles['react-spectrum-ListViewItem-draghandle-button']}>
+                  <DragHandle />
+                </div>
+              </div>
+              {showCheckbox &&
+                <Checkbox
+                  isSelected={isSelected}
+                  UNSAFE_className={listStyles['react-spectrum-ListViewItem-checkbox']}
+                  isEmphasized
+                  aria-label={checkboxProps['aria-label']} />
+              }
+              <SlotProvider
+                slots={{
```

Comment:
@reidbarber correct me if I'm wrong but I think this is a placeholder preview and won't be the final default one right? That might mean some of the stuff below might change/drift from the ListViewItem's appearance but if we aren't expecting anything super drastic then we can for sure move this out into a separate file

Reply:
I tried sharing these (it would be easier if the render preview was just ListViewItem itself), but it wasn't rendering properly for some reason. It would definitely be cleaner if it lived in a separate file though.

And @LFDanLu is right, we still need designs for render previews (for single and multiple items), so this is temporary.